util/asio/io_context_pool.h

```diff
@@ -178,6 +211,7 @@ class IoContextPool {
 
   typedef ::boost::asio::executor_work_guard<IoContext::io_context::executor_type> work_guard_t;
 
+  std::vector<int> cpu_idx_arr_;
```

Comment:
size_t

Reply:
Where do you use cpu_idx_arr?

Reply:
https://github.com/romange/gaia/pull/51/files#diff-9d369e4e94db23eeb0c6e80e14f1c711L63